app/assets/javascripts/student_profile/page_container.js

```diff
@@ -114,8 +114,8 @@
       });
     },
 
-    onSaveServiceFail: function(request, status, message) {
-      this.setState({ requests: merge(this.state.requests, { saveService: 'error' }) });
+    onSaveServiceFail: function(response, status, message) {
+      this.setState({ requests: merge(this.state.requests, { saveService: response.responseJSON.errors[0]}) });
```

Comment:
I was a bit hesitant on that bit. I'm not a fan of magic numbers, and also it's not really intuitive at all. I think we can get a lot more mileage out of api.js to have some more consistent return values.